packages/client/src/apis/kiosk.ts

```diff
@@ -1,11 +1,12 @@
 import { axiosInstance } from './util'
+import { TCategory } from 'types'
 
-export const getAllInfo = async () => {
+export const getAllInfo = async (): Promise<TCategory[] | undefined> => {
   try {
     const data = await axiosInstance.get('')
 
     if (data.status === 200) {
-      return data.data
+      return data.data.categories
```

Comment:
axios 인스턴스에서 반환하는 값은 네이밍을 response로 하는 건 어떨까요? 가독성이 좋아질 것 같습니다!
```suggestion
      return response.data.categories
```